webapp/src/images/icons/dot.jsx

```diff
@@ -2,22 +2,8 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import {DotIcon} from '@primer/octicons-react';
 
-export default class DotIcon extends React.PureComponent {
-    render() {
-        return (
-            <svg
-                viewBox='0 0 8 16'
-                version='1.1'
-                width='8'
-                height='16'
-                role='img'
-            >
-                <path
-                    fillRule='evenodd'
-                    d='M0 8c0-2.2 1.8-4 4-4s4 1.8 4 4-1.8 4-4 4-4-1.8-4-4z'
-                />
-            </svg>
-        );
-    }
-}
+export default () => (
+    <DotIcon size='small'/>
```

Comment:
This should be https://primer.style/octicons/dot-fill-16

![Screenshot from 2020-10-01 14-36-32](https://user-images.githubusercontent.com/16541325/94809886-89df9680-03f3-11eb-9150-53dd41a5a1dd.png)